src/Zend/Config/Writer/Xml.php

```diff
@@ -74,6 +74,10 @@ public function render()
 
         $xmlString = $dom->saveXML();
 
+        if ($xmlString === false) {
+            throw new Zend_Config_Exception('Error saving to XML string');
```

Comment:
This is the same Exception thrown earlier in this method, just trapping a possible `false` value on the return of `saveXML`.